torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1370,12 +1381,52 @@ Tensor computeConv2d(
   return Tensor(ResultBuf.node(), s);
 }
 
+Tensor computePrepackedConv2dClampRun(
```

Comment:
Not for this PR, but it would nice to have all these `compute*` for external calls in a separate file, may be under `operators/` folder.